### PR TITLE
[GD-15] fixing lambda timeouts when filtering keys

### PR DESCRIPTION
**Pull request by tvaintrob**

### CODE REVIEW

File: src/trace_senders/http.js [hunk 1]
```diff
@@ -19,7 +19,10 @@ const session = axios.create({
 
 module.exports.sendTrace = function sendTrace(traceObject) {
     utils.debugLog(`Posting trace to ${config.getConfig().traceCollectorURL}`);
-    utils.debugLog(`trace: ${JSON.stringify(traceObject, null, 2)}`);
+
+    if ((process.env.EPSAGON_DEBUG || '').toUpperCase() === 'TRUE') {
```
Comment (haimrait) [inline src/trace_senders/http.js:23]: why did you add the condition?
is the condition is not part of debugLog?

File: src/tracer.js [hunk 2]
```diff
@@ -21,6 +21,7 @@ const logSender = require('./trace_senders/logs.js');
 const httpSender = require('./trace_senders/http.js');
 
 const FILTER_TRACE_MAX_DEPTH = 50;
+const MAX_EVENTS = 30;
```
Comment (haimrait) [inline src/tracer.js]: change it to be an env variable

File: src/trace_senders/http.js (hunk 1 above)
Comment (tvaintrob) [reply src/trace_senders/http.js:23]: to avoid stringify when not in debug mode

File: src/tracer.js (hunk 2 above)
Comment (tvaintrob) [reply src/tracer.js]: done

**Review by haimrait — DISMISSED** [review]

**Review by haimrait — APPROVED** [review]
